### ISSUE TITLE
Inference result not shown

### ISSUE THREAD

**Issue by tfuru-macnica** [NVIDIA-ISAAC-ROS/isaac_ros_pose_estimation#13]
Hi.
I have just tried the quickstart demo. But I have not seen any result from the demo in RViz2.  "ros2 topic echo /poses" output empty data for the pose. Would you advise me how to fix this issue.

```
---
header:
  stamp:
    sec: 0
    nanosec: 0
  frame_id: camera
poses: []
---
```

In the first terminal (in which the inference pipeline was running), I have seen the following warning messages:
```
[component_container_mt-1] [WARN] [1676440195.245494673] [dope_decoder]: [NitrosPublisherSubscriberBase] Failed to get timestamp from a NITROS message (eid=182528)
[component_container_mt-1] [WARN] [1676440195.332820066] [dope_decoder]: [NitrosPublisherSubscriberBase] Failed to get timestamp from a NITROS message (eid=182558)
[component_container_mt-1] [WARN] [1676440195.452075203] [dope_decoder]: [NitrosPublisherSubscriberBase] Failed to get timestamp from a NITROS message (eid=182588)

```
The following is the platform which I am using:
```
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 510.85.02    Driver Version: 510.85.02    CUDA Version: 11.6     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  NVIDIA A40-24Q      On   | 00000000:02:02.0  On |                    0 |
| N/A   N/A    P0    N/A /  N/A |    841MiB / 24576MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
```
On the same machine, the Isaac ROS Object Detection is running fine.
 
Thanks.

**Comment by tfuru-macnica** [comment]
The DOPE demos have not been worked on my environment, but the CenterPose demo has been worked fine.

| Example Application | My Status |
| ------------- | ------------- |
| [Quickstart](https://github.com/NVIDIA-ISAAC-ROS/isaac_ros_pose_estimation#quickstart) | NG |
| [Tutorial for DOPE Inference](https://github.com/NVIDIA-ISAAC-ROS/isaac_ros_pose_estimation/blob/main/docs/dope-triton.md) | NG |
| [Tutorial for CenterPose Inference using Triton](https://github.com/NVIDIA-ISAAC-ROS/isaac_ros_pose_estimation/blob/main/docs/centerpose.md) | OK |

**Comment by tfuru-macnica** [comment]
On my system, the Ketchup DOPE model does not work, but the Milk DOPE model works fine.

**Comment by jaiveersinghNV** [comment]
Thanks for reporting this - we've reproduced the error on our end and we're looking into it.

**Comment by HedgeHao** [comment]
Any  updates or workaround? got the same error.

**Comment by jaiveersinghNV** [comment]
Hi @HedgeHao @tfuru-macnica , we recently merged a change with #23 that resolved the lack of detections on our end. 

Could you please try pulling the latest version and testing again? If you're still encountering problems, please let us know exactly what input data/model you used and what you're seeing as output.

**Comment by tfuru-macnica** [comment]
@jaiveersinghNV , I have confirmed that this issue has been resolved. Now I can run the Ketchup DOPE model. Thank you very much.
